src/components/organisms/Navbar/NavbarLink/NavbarLink.tsx

```diff
@@ -2,11 +2,11 @@ import React, { FC } from 'react';
 import styled from 'styled-components';
 import { colors } from '../../../../constants/colors';
 import Chevron from '../../../icons/Chevron/Chevron';
-import Link from '../../../atoms/Link/Link';
+import Link, { ILinkProps } from '../../../atoms/Link/Link';
 
 export interface IStyledNavbarLinkProps extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
   active: boolean;
-  color: string;
+  color?: ILinkProps['color'];
```

Comment:
Why not `IStyledNavbarLinkProps extends ILinkProps` ? and then you can get rid of this

Reply:
Hmm good point 🤔 , I'll give it a go 👍🏻